tests/php/StoreApi/Routes/ProductAttributes.php

```diff
@@ -53,7 +53,7 @@ public function test_get_items() {
 		$data     = $response->get_data();
 
 		$this->assertEquals( 200, $response->get_status() );
-		$this->assertEquals( 2, count( $data ) );
+		$this->assertEquals( count( wc_get_attribute_taxonomies() ), count( $data ) );
```

Comment:
Is it necessary? Isn't it better to hardcode the value?

Reply:
I was getting mismatched results between dev and CI (most probably because wp-env isn't stateless between steps or even jobs it seems 🙄)

Comparing the result coming from the API with wc_get_attribute_taxonomies() seems more robust since those should walk hand to hand. If we hardcode 2, but the count from taxonomies returns 6, and the test passes, would you say we have good code? Or if both return the same value but a wrong value, would you say it is our code's fault? 🤔